### PR TITLE
[DOCFIX]Add java doc for PermissionCheckTest

### PR DESCRIPTION
**Pull request by LiemLin**
Add java doc for PermissionCheckTest

### CODE REVIEW

**Comment by LiemLin** [comment]
alluxio-bot, check this please

File: core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java [hunk 1]
```diff
@@ -441,6 +441,13 @@ public void renameFailByDst() throws Exception {
     verifyRename(TEST_USER_1, TEST_DIR_FILE_URI, "/fileRenamed");
   }
 
+  /**
+   * verifyRename
+   * @param user
```
Comment (yupeng9) [inline core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java]: Can you add the descriptions for each field?

File: core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java [hunk 2]
```diff
@@ -441,6 +441,13 @@ public void renameFailByDst() throws Exception {
     verifyRename(TEST_USER_1, TEST_DIR_FILE_URI, "/fileRenamed");
   }
 
+  /**
+   * verifyRename
```
Comment (yupeng9) [inline core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java]: Per check style, you need a dot at the end.

File: core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java [hunk 3]
```diff
@@ -441,6 +441,13 @@ public void renameFailByDst() throws Exception {
     verifyRename(TEST_USER_1, TEST_DIR_FILE_URI, "/fileRenamed");
   }
 
+  /**
+   * verifyRename.
```
Comment (gpang) [inline core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java]: Could you describe the method as well as the parameters? Here is an example: https://github.com/Alluxio/alluxio/blob/v1.4.0/core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java#L1109

**Comment by yupeng9** [comment]
@LiemLin Any updates on this PR?

**Comment by yupeng9** [comment]
@LiemLin Feel free to reopen this PR after you address the comments.

**Pull request closed without merging**